driver-core/src/main/java/com/datastax/driver/core/NettyOptions.java

```diff
@@ -89,7 +89,9 @@
      *
      * @param threadFactory The {@link ThreadFactory} to use when creating a new {@code EventLoopGroup} instance;
      *                      The driver will provide its own internal thread factory here.
-     *                      It is safe to ignore it and use another thread factory.
+     *                      It is safe to ignore it and use another thread factory. Note however that for optimal
+     *                      performance it is recommended to use a factory that returns
+     *                      {@link io.netty.util.concurrent.FastThreadLocalThread} instances.
```

Comment:
It would be helpful to indicate `io.netty.util.concurrent.DefaultThreadFactory` as an example, so users know how to create a thread factory this way.  Although `NettyOptions` is for advanced users, so they might already know this.